modules/gdscript/gdscript_compiler.cpp

```diff
@@ -1955,10 +1954,9 @@ Error GDScriptCompiler::_parse_block(CodeGen &codegen, const GDScriptParser::Sui
 					// Add locals in block before patterns, so temporaries don't use the stack address for binds.
 					List<GDScriptCodeGenerator::Address> branch_locals = _add_block_locals(codegen, branch->block);
 
-#ifdef DEBUG_ENABLED
 					// Add a newline before each branch, since the debugger needs those.
```

Comment:
```suggestion
					// Add a newline before each branch, since the debugger needs those.
					// Also needed when tracking call stacks is enabled in the project settings.
```

Or remove both comments since we don't add them to every `write_newline()` call.